src/cbuild/util/zig_build.py

```diff
@@ -0,0 +1,54 @@
+def build(pkg, extra_args=[]):
+    profile = pkg.profile()
+    sysroot = profile.sysroot
+    with open(pkg.cwd / "cbuild_zig_libc.txt", mode="w") as f:
+        f.write(
+            f"""include_dir={sysroot}/usr/include
+sys_include_dir={sysroot}/usr/include
+crt_dir={sysroot}/lib
+msvc_lib_dir=
+kernel32_lib_dir=
+gcc_dir=
+
+"""
+        )
+
+    zig_arch = None
+    zig_cpu = None
+    match profile.arch:
+        # TODO other architectures
+        case "x86_64" | "aarch64":
+            zig_arch = profile.arch
+            zig_cpu = "baseline"
+
+    # The Zig build system only has a single install step, there is no
+    # way to build artifacts for a given prefix and then install those artifacts
+    # to that prefix at some later time. Therefore, we build and install to the zig-out
+    # directory and later copy the artifacts to the destdir in do_install().
+    # We use zig-out to avoid path conflicts as it is the default install
+    # prefix used by the zig build system.
+    pkg.do(
+        "zig",
+        "build",
+        "-j" + str(pkg.make_jobs),
+        "--sysroot",
+        sysroot,
+        "--libc",
+        "cbuild_zig_libc.txt",
+        "--search-prefix",
+        "/usr",
+        "--prefix",
+        "/usr",
+        "--system",
+        sysroot / "usr/src/zig/packages",
+        "--release=safe",
```

Comment:
you probably know this already but i assume this is meant to be a variable like zig_release_target= or similar (or at least i imagine hypothetically something might want to pick another option than release-safe)

Reply:
Note that this actually gives first choice to the upstream application author, and "safe" here is the fallback choice if the upstream author does not declare a default release mode. In any case, your point stands.

Reply:
ah, i didn't know that. that's good to know, and probably makes this a little less meaningful :)

Reply:
In any case, templates may override this default by adding `--release=fast` or whatever to `zig_build_args`.